File: python/problem6.py
# https://projecteuler.net/problem=6


if __name__ == "__main__":
    N = 100
    sum_of_squares = sum([n**2 for n in range(1, N+1)])
    square_of_sum = sum([n for n in range(1, N+1)])**2
    res = square_of_sum - sum_of_squares

    # result
    print(res)

File: matlab/problem6.m
clear; close all; clc
%% https://projecteuler.net/problem=6

N = 100;

% measure execution speed
tic
sum_of_squares = sum((0:N).^2);
square_of_sum = sum(0:N)^2;
res = square_of_sum - sum_of_squares;
toc
% Elapsed time is 0.000231 seconds.

% result
disp(res)

